adagrams/game.py

```diff
@@ -1,11 +1,127 @@
+import random
 def draw_letters():
-    pass
+    letters =[]
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    while len(letters) <10:
+        letter = random.choice(list(LETTER_POOL)) 
+        if LETTER_POOL[letter] > 0:
+            LETTER_POOL[letter] -= 1
+        else: 
```

Comment:
One suggestion - rather than have an `else` block that skips the append operation on line 39, why not just append within the `if` block?